pcsx2/ImGui/ImGuiFullscreen.h

```diff
@@ -276,9 +284,10 @@ namespace ImGuiFullscreen
 	void SetNotificationVerticalPosition(float position, float direction);
 	void SetNotificationPosition(float horizontal_position, float vertical_position, float direction);
 
-	void OpenBackgroundProgressDialog(const char* str_id, std::string message, s32 min, s32 max, s32 value);
-	void UpdateBackgroundProgressDialog(const char* str_id, std::string message, s32 min, s32 max, s32 value);
-	void CloseBackgroundProgressDialog(const char* str_id);
+	ImGuiID GetProgressDialogID(const char* str_id);
```

Comment:
Make `GetProgressDialogID()` static.
that will fix the compile error you got before.

Reply:
Ahhhhh alright I'll go fix these real quick then